### DISCUSSION TITLE
Using this with a WT32-ETH01

### DISCUSSION THREAD

**Discussion by ChESch** [stas-sl/esphome-sound-level-meter#32] [Q&A]
Hello there!

I would love to use this project with a WT32-ETH01 with Ethernet - but when I want to compile, esphome does not like that the GPIO 23 and 18 would be used twice.
Is there a solution for this?

![image](https://github.com/user-attachments/assets/55517db8-b5d2-48f1-af3b-7efd405b5511)
![image](https://github.com/user-attachments/assets/a57cca5e-3d4d-43bf-8402-8b5b384006fa)

Thanks!

**Comment by stas-sl** [comment]
You can just use any other free pins on your board

**Comment by ChESch** [comment]
This worked, thank you very much!

I have another question, I am using this board: https://www.pschatzmann.ch/home/2021/11/17/using-a-i2s-adc-audio-i2s-capture-card-module-with-an-esp32/
It seems to be working, but when there is absolutly no Audio playing, I am getting values of arround 87 db, which seems rather high

My goal is just to get an ON/OFF Switch from a threshold, so that might be enough for me, but I am still curious why the values are so high 🤔 

![image](https://github.com/user-attachments/assets/777329db-2946-4b0f-85cf-933cb3a2bf29)
